### ISSUE TITLE
Documentation Checklist: -ansible-core 2.15 release

### ISSUE THREAD

**Issue by samccann** [ansible/ansible#80452]
### Summary

Documentation checklist to prepare for and deliver the ansible-core 2.15 release. See follow-on comment for the full checklist.

### Issue Type

Documentation Report

### Component Name

docs/docsite/sphinx_conf/ansible_conf.py

### Ansible Version

```console
$ ansible --version
2.15
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by ansibot** [comment]
Files identified in the description:
* [`docs/docsite/sphinx_conf/ansible_conf.py`](https://github.com/ansible/ansible/blob/devel/docs/docsite/sphinx_conf/ansible_conf.py)

If these files are incorrect, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/devel/ISSUE_HELP.md)

**Comment by samccann** [comment]
Tasks include:

- [ ] Review changelogs in stable-2.15 for ansible-core
- [ ] Review ansible-core porting guide and ensure it links to core 2.15 changelogs
- [ ] include ansible-core in Release Status Grid and ensure maintenance/eol releases listed accurately. Include links to changelogs.
- [ ] Update core_conf.py:
    - [ ] Set 'latest_version': '2.15 ', under html_context
    - [ ] Update ‘available_versions’ to 2.15, 2.14, 2.13,  devel, under ‘html_context’.
    - [ ] Merge to `devel`
    - [ ] Backport to `stable-2.15`, `stable-2.14`, and `stable-2.13`
    - [ ] Verify Ansible package porting guide and roadmaps are listed under ‘exclude_patterns’. 
    - [ ] In `stable-2.15` branch - change VERSION (capitalized) from `devel` to `2.15`.

- [ ] publish on release day.
- [ ] Translation docs for core (this usually lags some time so may need to open separate issue to track):
    - [ ] test publish core-2.15 with japanese language selection in Jenkins job. 
    - [ ] Update `core_lang_conf.py:
       - [ ]  - Ensure `exclude patterns' match core_conf.py
       - [ ]  Set 'latest_version': '2.15', under html_context
      - [ ] Update ‘available_versions’ to 2.15_ja, 2.14_ja, 2.13_ja,  devel, under ‘html_context’.
      - [ ] Merge to `devel`
      - [ ] Backport to `stable-2.15`, `stable-2.14`, and `stable-2.13`
      - [ ]  In `stable-2.15` branch - change VERSION (capitalized) from `devel` to `2.15_ja`.
    - [ ] publish to prod.
    - [ ] Update the docsite to include the new translation 

Post release:
- [ ] Update the backport instructions to target the new “latest” branch, see https://github.com/ansible/ansible/pull/56578 
- [ ] Make sure all server-side redirects are updated as necessary - see https://github.com/ansible/docsite/pull/11 
- [ ] ~Swiftype updates~ - swiftype does not index /ansible-core/ at all at this point so these steps do not apply
  - [ ] ~Update Swiftype Search engine for ansible-core to blacklist the EOL release (/ansible-core/2.11)~  
  - [ ] ~Kick off a reindex on the swiftype engine~
- [ ] End-of-life the 2.12 docs
   - [ ] In the EOL branch (stable-2.12), update page banner to EOL to the release no longer supported. See https://github.com/ansible/ansible/pull/76669/files

**Comment by oraNod** [comment]
@samccann Hi, just giving an update on the state of this checklist. I've merged the version switcher to devel and backported to stable-2.15. I also changed VERSION=2.15 on the stable-2.15 branch.

I believe there were some scheduling conflicts that mean that 2.15 is released ahead of other stable backports. So stable-2.14 and stable-2.13 will get merged next week.

I published devel and stable-2.15 to test and verified and then published to production. Everything seems OK. Obvs the drop-down on 2.14 and 2.13 is out of sync but we just have to wait on those backports.

**Comment by samccann** [comment]
cool thanks!

**Comment by oraNod** [comment]
@samccann stable-2.13 and stable-2.14 docs are now published
